tests/asset/AssetPanelTest.php

```
@@ -264,6 +311,88 @@ public function testIsEnabledTrueWhenAssetManagerResolves(): void
+        self::assertSame(
+            1,
+            $entry['imports'] ?? null,
+            'Import count must be captured.',
+        );
+        self::assertTrue(
+            $entry['isEntry'] ?? null,
+            'Entry flag must be captured.',
+        );
+
+        @unlink($manifestPath);
+    }
```

PHPMD (2.15.0)

[error] Remove error control operator '@' on line 318. (undefined)

(ErrorControlOperator)